projects/ng-rule-editor/tsconfig.spec.json

```diff
@@ -3,8 +3,13 @@
   "extends": "../../tsconfig.json",
   "compilerOptions": {
     "outDir": "../../out-tsc/spec",
+    "moduleResolution": "node",
     "types": [
       "jasmine"
+    ],
+    "typeRoots": [
+      "node_modules/@types",
+      "node_modules/@types/node"
```

Comment:
Can you revert the changes back please, it was meant for debugging.

Reply:
Fixed